web/src/components/molecules/Common/ResizableProTable/index.tsx

```diff
@@ -0,0 +1,87 @@
+import styled from "@emotion/styled";
+import { useEffect, useState } from "react";
+
+import ProTable, {
+  ProColumns,
+  OptionConfig,
+  ListToolBarProps,
+} from "@reearth-cms/components/atoms/ProTable";
+import { ResizableTitle } from "@reearth-cms/components/molecules/Common/ResizableProTable/resizble";
+import type { ResizeCallbackData } from "@reearth-cms/components/molecules/Common/ResizableProTable/resizble";
+
+export type Props = {
+  className?: string;
+  dataSource?: any[];
+  proColumns?: ProColumns<any, "text">[];
+  loading: boolean;
+  options?: false | OptionConfig | undefined;
+  toolbar?: ListToolBarProps | undefined;
+};
+
+const ResizableProTable: React.FC<Props> = ({
+  dataSource,
+  proColumns,
+  loading,
+  options,
+  toolbar,
+}) => {
+  const [columns, setColumns] = useState<ProColumns<any, "text">[]>([]);
+
+  useEffect(() => {
+    if (Array.isArray(proColumns)) {
+      setColumns(proColumns);
+    }
+  }, [proColumns, setColumns]);
```

Comment:
Is there a specific reason you are checking if it is an array and not just checking if it exists (if(proColumns))?

Reply:
@KaWaite , first render this proColumns may be `undefined`, maybe I can check it like this: `if (!proColumns) return`?

Reply:
I think if it can be undefined, doing `if(!!proColumns)` will suffice

Reply:
@KaWaite , done. `if(proColumn)` seems to be enough.